core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1064,7 +1099,9 @@ private ExecutionModel parseInsert(GenericLexer lexer) throws SqlException {
             throw SqlException.$(lexer.lastTokenPosition(), "'into' expected");
         }
 
-        model.setTableName(expectLiteral(lexer));
+        tok = tok(lexer, "table name");
+
+        model.setTableName(nextLiteral(GenericLexer.assertNoDotsAndSlashes(GenericLexer.unquote(tok), lexer.lastTokenPosition()), lexer.lastTokenPosition()));
```

Comment:
We have `TableUtils.isValidColumnName` and it's more strict than just dots and slashes. I believe same level of validation should be for table names as it is for column names

Reply:
we need to tidy this up separately, the above is table name - column name requirements are different